packages/compose/src/hooks/use-dragging/index.js

```diff
@@ -8,6 +8,12 @@ import { useCallback, useEffect, useRef, useState } from '@wordpress/element';
  */
 import useIsomorphicLayoutEffect from '../use-isomorphic-layout-effect';
 
+/**
+ * @param {Object} props
+ * @param {(e: MouseEvent) => void} props.onDragStart
```

Comment:
I'm assuming we intentionally use `MouseEvent ` instead of `DragEvent`?

Reply:
Yeah, `onDragStart` gets passed to `onmousedown` instead of `ondrag` actually.